tests/e2e/cucumber/features/smoke/fullTextSearch.ocis.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Search
+  As a user
+  I want to do full text search
+  So that I can find the files with the content I am looking for
+
+  Background:
+    Given "Admin" creates following users using API
+      | id    |
+      | Alice |
+      | Brian |
+    And "Admin" sets the default folder for received shares to "Shares"
+    And "Admin" disables share auto accepting
```

Comment:
Aren't these options default in ocis? or is this a feature for both oc10 aond ocis if so we need to change the file name
```suggestion
```

Reply:
True.